doc/RecordingControl.xml

```diff
@@ -1684,8 +1787,420 @@ SetRecordingJobMode(JobToken, Active)
+      </para>
+    </section>
+    <section xml:id="_refCloudRecordingRecordingFormatMP4">
+      <title>Recording format MP4</title>
+      <para>
+        If the <literal>Format</literal> configured for a recording is <literal>MP4</literal>, the device shall follow the rules given in this section.
+      </para>
+      <para>
+        The device shall open a single span for an active recording containing all tracks of the recording.
+        The device shall use <literal>mp4</literal> as the extension for this span type.
+        The device shall upload each segment object belonging to a span of this type with the content type <literal>video/mp4</literal>.
+        The device should include the <literal>codecs</literal> parameter in the content type according to RFC 6381.
+      </para>
+      <para>
+        The <literal>MovieBox</literal> contained in a segment shall not reference any samples.
+        A segment shall contain a <literal>MovieFragmentRandomAccessBox</literal> as the last box.
+        The <literal>MovieFragmentRandomAccessBox</literal> shall provide references for all tracks and their applicable <literal>MovieFragmentBox</literal> instances.
+      </para>
+    </section>
+    <section xml:id="_refCloudRecordingRecordingFormatCMAF">
+      <title>Recording format CMAF</title>
+      <para>
+        If the <literal>Format</literal> configured for a recording is <literal>CMAF</literal>, the device shall follow the rules given in this section.
+      </para>
+      <para>
+        The device shall open up to three spans for an active recording, each containing a single track of the recording.
+      </para>
+      <para>
+        If the recording contains a video track, the device shall open a video span containing this track.
+        The device shall use <literal>m4v</literal> as the extension for this span type.
+        The device shall upload each segment object belonging to a span of this type with the content type <literal>video/mp4</literal>.
+      </para>
+      <para>
+        If the recording contains an audio track, the device shall open an audio span containing this track.
+        The device shall use <literal>m4a</literal> as the extension for this span type.
+        The device shall upload each segment object belonging to a span of this type with the content type <literal>audio/mp4</literal>.
+      </para>
+      <para>
+        If the recording contains a metadata track, the device shall open a metadata span containing this track.
+        The device shall use <literal>m4m</literal> as the extension for this span type.
+        The device shall upload each segment object belonging to a span of this type with the content type <literal>application/mp4</literal>.
+      </para>
+      <para>
+        The device should include the <literal>codecs</literal> parameter in the content type according to RFC 6381 for all span types defined in this section.
```

Comment:
does CMAF format have any limitation on supported video codecs( i.e H.264/H.265/MJPEG/MPEG4) & audio codecs(i.e G711/G726/AAC)?

Reply:
As far as I'm concerned, CMAF does not limit codecs.